packages/components/src/hooks/useBusyState.test.ts

```diff
@@ -0,0 +1,57 @@
+import { controlledPromise } from '@terra-ui-packages/core-utils';
+import { act, renderHook } from '@testing-library/react';
+
+import { useBusyState } from './useBusyState';
+
+describe('useBusyState', () => {
+  it('sets busy to true and false during data call', async () => {
+    // Arrange
+    const [promise, controller] = controlledPromise<string>();
+    const getData: () => Promise<string> = jest.fn(() => promise);
+
+    // Act
+    const hookRender = renderHook(() => useBusyState());
+    const [isBusy1, withBusy1] = hookRender.result.current;
```

Comment:
_Super_ nit: more descriptive names would make the assertions easier to interpret.

```suggestion
    const [isBusyInitial, withBusy] = hookRender.result.current;
```

Reply:
agreed